CMakeLists.txt

```diff
@@ -51,6 +51,17 @@ set(CMAKE_CXX_EXTENSIONS NO)
 include(SwiftUtils)
 include(CheckSymbolExists)
 include(CMakeDependentOption)
+include(CheckLanguage)
+
+# Enable Swift for the host compiler build if we have the language. It is
+# optional until we have a bootstrap story.
+check_language(Swift)
+if(CMAKE_Swift_COMPILER)
+  enable_language(Swift)
+else()
+  message(STATUS "WARNING! Did not find a host compiler swift?! Can not build
+                  any compiler host sources written in Swift")
```

Comment:
```suggestion
  message(WARNING "Did not find a host compiler swift. Can not build any compiler host sources written in Swift")
```

Reply:
I think it may be nice to sink this lower to the point of use.

Reply:
Mind if I do that in a follow on?